### PR TITLE
idnits 2.14.01

### PR DESCRIPTION
**Pull request by larseggert**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/idnits.rb [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
   homepage "https://tools.ietf.org/tools/idnits/"
-  url "https://tools.ietf.org/tools/idnits/idnits-2.13.02.tgz"
-  sha256 "6e42b044c79dc4e616d10ee9e283c20acc741601811a6acfc0c0d310afdf0823"
+  url "https://tools.ietf.org/tools/idnits/idnits-2.14.01.tgz"
+  sha256 "5b49749b0e4dc610fae9c12780c7a4d02723a5f4b701271bdd374d909c186654"
 
   bottle :unneeded
 
   depends_on "aspell"
+  depends_on "languagetool"
```
Comment (mikemcquaid) [inline Formula/idnits.rb]: Can you elaborate a bit on why this is needed now? Thanks!

File: Formula/idnits.rb [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 
   def install
     bin.install "idnits"
   end
+
+  test do
+    system "idnits --version"
```
Comment (mikemcquaid) [inline Formula/idnits.rb]: `system "#{bin}/idnits", "--version"`. Ideally though this would run on an input file?

File: Formula/idnits.rb (hunk 1 above)
Comment (larseggert) [inline Formula/idnits.rb]: To enable the --grammarcheck option

File: Formula/idnits.rb (hunk 2 above)
Comment (larseggert) [inline Formula/idnits.rb]: Yes, but no valid input is included with the distribution. Is it OK to wget one during the test?

File: Formula/idnits.rb (hunk 1 above)
Comment (mikemcquaid) [inline Formula/idnits.rb]: Cool. Maybe worth making `:recommended` so users can opt-out?

File: Formula/idnits.rb (hunk 2 above)
Comment (mikemcquaid) [inline Formula/idnits.rb]: Ideally write one like https://github.com/Homebrew/homebrew-core/blob/master/Formula/cmake.rb#L65. Use `EOS.undent` to write a multiple line file.

Comment (larseggert) [inline Formula/idnits.rb]: A minimal valid I-D is several pages of text, however

Comment (mikemcquaid) [inline Formula/idnits.rb]: In that case making one a `resource` and downloading it with `.stage` maybe be better 👍

**Comment by UniqMartin** [comment]
Merged. Thank you for this contribution to Homebrew, @larseggert! :tada:

**Pull request merged**